docs/fx-core/switch-tenant-or-subscription-help.md

```diff
@@ -1,28 +1,28 @@
-This doc is to help you understand what will happen when provisioning in an already-provisioned environment but with different account or Azure subscription or local debugging again with another Microsoft 365(M365) account. We will also explain how to recover from the backups in this doc.
+This doc is to help you understand what will happen when provisioning in an already-provisioned environment but with different account or Azure subscription or local debugging again with another Microsoft 365(Microsoft 365) account. We will also explain how to recover from the backups in this doc.
 
```

Comment:
`Microsoft 365(Microsoft 365)` has duplicates.

Reply:
Thanks for catching this! Fixed.